arrow-libs/fx/arrow-fx-coroutines/src/main/kotlin/arrow/fx/coroutines/Fiber.kt

```diff
@@ -147,12 +161,18 @@ suspend fun <A> (suspend () -> A).forkScoped(
  *
  * @see ForkConnected for a fork operation that wires cancellation to its parent in a safe way.
  */
-// TODO provide proper deprecation annotation
+@Deprecated(
+  "Use Deferred with KotlinX Coroutines Structured Concurrency",
+  ReplaceWith("GlobalScope.async(ctx) { f() }", "kotlinx.coroutines.async", "kotlinx.coroutines.GlobalScope")
```

Comment:
same here, I wouldn't include GlobalScope here

```suggestion
  ReplaceWith("async(ctx) { f() }", "kotlinx.coroutines.async")
```

Reply:
Same issue. This has now completely different semantics as the function being deprecated.